controllers/mealplan.controller.js

```diff
@@ -0,0 +1,65 @@
+const MealPlan = require("../models/meal-plan.model");
+
+const {
+  getUserMealPlans,
+  getUserMealPlan,
+  udpateUserMealPlan,
+  deleteUserMealPlan,
+  createUserMealPlan,
+} = require("../services/userMealPlan.service");
+
+exports.user_mealplans_get_all = async (req, res, next) => {
+  userId = req.user._id;
+  try {
+    const mealPlans = await getUserMealPlans(userId);
+    res.status(200).json(mealPlans);
+  } catch (err) {
+    res.status(500).json({ message: err.message });

```

Comment:
instead of this line just do 'next(error);' service throws erros and controller should just pass it further. Let me know if postman wont throw these errors